grails-app/views/locationType/edit.gsp

```diff
@@ -46,7 +46,6 @@
                         class="value ${hasErrors(bean: locationTypeInstance, field: 'description', 'errors')}">
                         <g:select name="locationTypeCode" class="chzn-select-deselect"
                                   from="${org.pih.warehouse.core.LocationTypeCode.list()}"
-                                  noSelection="['': '']"
```

Comment:
Removing this property means that the first one from the list is picked automatically or how is it working right now?

Reply:
yeah that's right. The first element of LocationTypeCode.list(), which is DEPOT

Reply:
Thinking about this more, I would actually prefer the default to be empty and for the form submission to return and display a validation error for blank/null. 

Can you post a screenshot of the ugly page (I assume this is the exception stacktrace) so I can figure out the best way to handle it?